phoenicis-tools/src/main/java/org/phoenicis/tools/checksum/ChecksumCalculator.java

```diff
@@ -36,7 +36,7 @@
 @Safe
 public class ChecksumCalculator {
     private static final int BLOCK_SIZE = 2048;
-    private static final String WAIT_MESSAGE = tr("Please wait while we are verifying the file ...");
+    private static final String WAIT_MESSAGE = tr("Verifying the file…");
```

Comment:
See above.

Reply:
Perhaps just "Verifying…" would be better for consistency

https://transvision.mozfr.org/?recherche=Verifying%E2%80%A6&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings

Reply:
With "Verifying..." only it is not clear what's happening (i.e. validating the checksum of a file).

Reply:
done

Reply:
...